libraries/testing/tester.cpp

```diff
@@ -499,7 +499,7 @@ namespace eosio::testing {
          // wait for this node's vote to be processed
          size_t retrys = 200;
          while (!c.node_has_voted_if_finalizer(c.head_block_id()) && --retrys) {
-            std::this_thread::sleep_for(std::chrono::milliseconds(10));
+            std::this_thread::sleep_for(std::chrono::milliseconds(5));
```

Comment:
It would be better if `retries` and the sleep time are defined as constants so that they can be kept consistent in the future. But you don't need to make this change for now.